src/apps/tabs/components/__tests__/header.react-test.js

```diff
@@ -0,0 +1,30 @@
+jest.autoMockOff();
+
+describe('ProductGroup', function() {
+  let ProductGroup = require('../product-group.react');
+  let React = require('react/addons');
+  let TestUtils = React.addons.TestUtils;
+  let product = {
+    name: 'Name',
+    tagline: 'Tagline',
+    day: '2015-01-01',
+    discussion_url: 'http://example.com',
+    screenshot_url: { '300px': 'http://example.com/screen' },
+    votes_count: 32,
+    comments_count: 22
+  };
+
+  function html() {
+    return TestUtils.renderIntoDocument(
+      <ProductGroup products={[product]} />
+    ).getDOMNode().innerHTML;
+  }
+
+  it('renders the group name', function() {
+    expect(html()).toContain('Thursday');
```

Comment:
I was thinking that `TestUtils.renderIntoDocument(X)).getDOMNode().innerHTML` would be a great matcher like:

```javascript
expect(<ProductGroup products={[product]} />).toRenderInHtml(product.name)
```